envs/environment.yaml

```
@@ -25,7 +25,7 @@ dependencies:
 - xlrd
 - openpyxl
 - seaborn
-- snakemake-minimal<8
+- snakemake-minimal>=8
```

Maybe advisable to replace into:
```suggestion
- snakemake-minimal<8.25
```
There is an issue in snakemake that leads to issues in -eur, see https://github.com/PyPSA/pypsa-eur/blob/b6b18ad5f1d722fbe5227ef955be348947a63eac/envs/environment.yaml#L24

On the other hand, we can now lift the the limitation on matplotlib, it was linked to a limitation on the pypsa version that no longer applies likely